src/components/TextButton/TextButton.visual.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
+import { TextButton } from './';
+
+class TextButtonVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    const { mobile } = this.props;
+
+    return (
+      <TPAComponentsProvider value={{ mobile }}>
+        <TextButton data-hook={'storybook-e2e-TextButton'} {...this.props} />
+      </TPAComponentsProvider>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
+      },
+      {
+        it: 'mobile',
+        props: {
+          mobile: true,
```

Comment:
again `mobile` can be removed.
However we should add the different priorities to this test.
It's very important since these are the snapshot tests, that will ensure the (visual) api doesn't break.